WordPress/src/main/res/values/strings.xml

```diff
@@ -503,6 +507,8 @@
     <string name="site_settings_location_title">Enable Location</string>
     <string name="site_settings_default_category_title">Default Category</string>
     <string name="site_settings_default_format_title">Default Format</string>
+    <string name="site_settings_default_image_width_title">Default Image Width</string>
```

Comment:
This is (almost exactly) the same text as `max_thumbnail_px_width`. Please consolidate.